### DISCUSSION TITLE
Twitch login

### DISCUSSION THREAD

**Discussion by Valentin-Metz** [streamlink/streamlink-twitch-gui#792] [Q&A] [answered]
Has the option to log into twitch been removed?
I'm trying to automate twitchdrops, which are linked to an account.

**Comment by bastimeyer** [comment, marked as answer]
Wrong repo...
- https://github.com/streamlink/streamlink/blob/master/CHANGELOG.md#streamlink-130-2019-11-22
- https://github.com/streamlink/streamlink/issues/2680#issuecomment-557605851

**Reply by bastimeyer** [reply]
Related Streamlink Twitch GUI changelog:
https://github.com/streamlink/streamlink-twitch-gui/blob/master/CHANGELOG.md#v190-2019-11-23
